src/hooks/event-bus.ts

```
@@ -44,10 +51,25 @@ export class EventBus {
+  }
+
+  /**
+   * Hot-path optimized emit for MessageRouted events.
+   * Avoids rest/spread overhead of the generic `emit()`.
+   */
+  public emitMessageRouted(subject: string, kind: MessageKind): void {
+    const hook = this.hooks[TransportEvent.MessageRouted];
+
+    if (!hook) return;
+
+    this.callHook(TransportEvent.MessageRouted, hook as (...a: unknown[]) => unknown, subject, kind);
```

ESLint

[error] Replace `TransportEvent.MessageRouted,·hook·as·(...a:·unknown[])·=>·unknown,·subject,·kind` with `⏎······TransportEvent.MessageRouted,⏎······hook·as·(...a:·unknown[])·=>·unknown,⏎······subject,⏎······kind,⏎····`

(prettier/prettier)